azplugins/BondEvaluatorDoubleWell.h

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// This file is part of the azplugins project, released under the Modified BSD License.
+
+// Maintainer: astatt
+
+/*! \file BondEvaluatorDoubleWell.h
+    \brief Defines the bond evaluator class for a double well potential
+*/
+
+#ifndef AZPLUGINS_BOND_EVALUATOR_DOUBLE_WELL_H_
+#define AZPLUGINS_BOND_EVALUATOR_DOUBLE_WELL_H_
+
+#ifndef NVCC
+#include <string>
+#endif
+
+#include "hoomd/HOOMDMath.h"
+
+#ifdef NVCC
+#define DEVICE __device__
+#else
+#define DEVICE
+#endif
+
+namespace azplugins
+{
+namespace detail
+{
+
+//! Class for evaluating the double well bond potential
+/*!
+This bond potential follows the functional form
+\f{eqnarray*}
+V_{\rm{DW}}(r)  =   \frac{V_{min}}{b^4} \left[ \left( r - a/2 \right)^2 -b^2 \right]^2
+\f}
+and has two minima at r = 1/2(a +/- 2b), seperated by a maximum at a/2 of height V_max.
```

Comment:
Sorry, I missed flagging this as well earlier. Can you also change this to
```
r = (a +/- 2b)/2
```
or better
```
r = a/2 +/- b
```
here, below, and in `bond.py` for readability?